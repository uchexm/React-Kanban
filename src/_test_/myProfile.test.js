import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Myprofile from '../components/myProfile/myProfile';

it('Test if MyProfile rendered', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Myprofile />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
