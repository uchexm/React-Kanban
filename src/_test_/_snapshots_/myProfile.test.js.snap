// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if MyProfile rendered 1`] = `
<div
  style={
    Object {
      "display": "flex",
    }
  }
>
  <article
    className="rocketProfileContainer"
  >
    <h1
      className="title"
    >
      My Mission
    </h1>
    <p>
      <em>
        You don't have any missions joined at the moment
      </em>
    </p>
  </article>
  <article
    className="rocketProfileContainer"
  >
    <h1
      className="title"
    >
      My Rockets
    </h1>
    <p>
      <em>
        You don't have any reserves at the moment
      </em>
    </p>
  </article>
</div>
`;